documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -312,49 +232,32 @@ The IOTA Identity framework currently supports two Verification Method Types:
+All private keys or seeds used for the `did:iota` method should be equally well protected by the users. Private keys of the state controller and the governor are especially important as they control how keys are added or removed, providing full control over the identity. The IOTA Identity framework utilizes the [Stronghold project](https://github.com/iotaledger/stronghold.rs), a secure software implementation isolating digital secrets from exposure to hacks or leaks. Developers may choose to add other ways to manage the private keys in a different manner.
 
 ## Privacy Considerations
 
 ### Personal Identifiable Information
 
-The public IOTA Tangle networks are immutable networks. This means that once something is uploaded, it can never be completely removed. That directly conflicts with certain privacy laws such as GDPR, which have a 'right-to-be-forgotten' for Personal Identifiable Information (PII). As such, users should NEVER upload any PII to the Tangle, including inside DID Documents. The IOTA Identity framework allows Verifiable Credentials to be published to the Tangle directly, however this feature should only be utilized by Identity for Organisation and Identity for Things. 
+The public IOTA Tangle networks are immutable networks. This means that once something is uploaded, it can never be completely removed. For example, destroying an Alias Output will remove it from the ledger state, but it can still be stored in permanodes or by any party that records historical ledger states.
+
+That directly conflicts with certain privacy laws such as GDPR, which have a 'right-to-be-forgotten' for Personal Identifiable Information (PII). As such, users should NEVER upload any PII to the Tangle, including inside DID Documents. The IOTA Identity framework allows Verifiable Credentials to be published to the Tangle directly, however this feature should only be utilized by Identity for Organisation and Identity for Things.
 
 ### Correlation Risks
 
-As with any DID method, identities can be linked if they are used too often and their usage somehow becomes public. IOTA provides a simple solution for this problem: as creating identities is completely feeless, it is recommended to make new identities on a per interaction basis (Pairwise DIDs). The IOTA Identity framework will provide support for this in the future. 
+As with any DID method, identities can be linked if they are used too often and their usage somehow becomes public. Additionally, a DID can be correlated with funds if the Alias Output used to store the DID Document or any of its controllers is used for holding, transferring or controlling coins or NFTs. 
```

Comment:
> As with any DID method, identities can be linked if they are used too often and their usage somehow becomes public. 

I know you didn't write this, but this should be clarified. Currently, it's barely understandable what it's supposed to mean, and how one is supposed to use a DID to avoid or mitigate this problem.

Reply:
It should link out to the W3C specification section, which has some advice on how to mitigate it: https://www.w3.org/TR/did-core/#did-correlation-risks

Reply:
Thanks Criag, that was my idea too.